src/Nox.Types.EntityFramework/Types/StreetAddress/StreetAddressDatabaseConfigurator.cs

```diff
@@ -22,29 +21,47 @@ public class StreetAddressDatabaseConfigurator : INoxTypeDatabaseConfigurator
             x =>
             {
                 x.Ignore(nameof(StreetAddress.Value));
+
```

Comment:
we could get the compoundtype attribute and use it to define the IsRequired so we do not need to change in two places... because we will forget, do you think the effort is worth?

Reply:
Yes, I agree. Changes were applied.